### PR TITLE
Add Seamonkey to Readme.md

### PR DESCRIPTION
**Pull request by polarbearing**
Mention Seamonkey in the Readme, and that it has to be installed manually.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ irc.mozilla.org.
 
 ### Browser Extensions
 
-#### Firefox
+#### Firefox and Seamonkey
 
-PDF.js is built into version 19+ of Firefox, however one extension is still available:
+PDF.js is built into version 19+ of Firefox, however one extension is still available, in Seamonkey it needs to be manually installed:
```
Comment (yurydelendik) [inline README.md]: It's not clear what "it needs to be manually installed" mean. I don't think we need to add Seamonkey in Readme.md. However we can add description at https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-extensions if you explain installation process.

Comment (polarbearing) [inline README.md]: In firefox it is built in nowadays. It is not an installable extension. In Seamonkey, I load the URL given below in line 36, and it installs the xpi file as an extension, which I can then view in about:addons.
If you mention Chrome and Opera in this readme, why not Seamonkey?

Comment (yurydelendik) [inline README.md]: > why not Seamonkey?

Good question.. Probably because we don't know if it works there... and lack of testing on Seamonkey.

**Comment by timvandermeij** [comment]
How about we keep the README as it currently is, but below the "Development version" link and before the Chrome/Opera section we add one line that says "The extension has also been reported to be compatible with Seamonkey, but support for Seamonkey is not guaranteed by the developers."

That way we do mention Seamonkey, but also that we do not use it for testing so we can't guarantee that it will always work.

**Comment by polarbearing** [comment]
Fine for me, as long as Seamonkey appears in the context of pdf.js thus searchengines find it.

Currently when searching the viewer from within the Seamonkey addon system, it only finds the outdated version 1.0.277.1 here ```https://addons.mozilla.org/en-US/seamonkey/addon/pdfjs/?src=search```, and that attracts stupid reviews.

**Comment by timvandermeij** [comment]
@polarbearing Could you update this PR with the mentioned change above? Be sure to squash the commits if you have multiple after the change (see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits on how to do that easily).

**Comment by timvandermeij** [comment]
Merged in #6665. Thank you.

**Comment by polarbearing** [comment]
Thanks for merging. BTW, squashing commits becomes a major hassle if you are not in a development workflow anyway and just making a quick change with the online editor.

**Comment by timvandermeij** [comment]
@polarbearing Yes, we thought that that might be an issue here too, hence we did that for you. Thank you for the patch!

**Pull request closed without merging**